src/PKSim.Core/Model/IndividualMolecule.cs

```diff
@@ -24,26 +25,35 @@ public virtual double OntogenyFactor
          set => OntogenyFactorParameter.Value = value;
       }
 
-      public IParameter OntogenyFactorParameter => this.Parameter(CoreConstants.Parameters.ONTOGENY_FACTOR);
+      public IParameter OntogenyFactorParameter => this.Parameter(ONTOGENY_FACTOR);
 
       public virtual double OntogenyFactorGI
       {
          get => OntogenyFactorGIParameter.Value;
          set => OntogenyFactorGIParameter.Value = value;
       }
 
-      public IParameter OntogenyFactorGIParameter => this.Parameter(CoreConstants.Parameters.ONTOGENY_FACTOR_GI);
+      public IParameter OntogenyFactorGIParameter => this.Parameter(ONTOGENY_FACTOR_GI);
+
+      public IParameter DiseaseFactorParameter => this.Parameter(DISEASE_FACTOR);
+
+      public virtual double DiseaseFactor
```

Comment:
This parameter can be accessed directly in the different implementation to update the value based on the disease state if required.